app/views/hold_recalls/_header.html.erb

```diff
@@ -0,0 +1 @@
+<h1 id='dialogTitle'>Request [checked-out | on-order] item</h1>
```

Comment:
I need to update this.  We don't have the tooling to know if the item is one or the other, so I need to build that in.